zabbix/README.md

```diff
@@ -0,0 +1,7 @@
+## [Zabbix](https://www.zabbix.com/) with PostgresSQL
+
+Change image postfix to `mysql` to use with MySQL.
```

Comment:
Did you mean "Change image postfix to 'postgres' to use with PostgreSQL"?

Reply:
In this compose file it is with Postgres db.